R/safetyGraphicsApp.R

```diff
@@ -14,7 +15,11 @@
 #'
 #' @export
 #'
-safetyGraphicsApp <- function(maxFileSize = NULL) {
+safetyGraphicsApp <- function(charts = NULL, maxFileSize = NULL) {
+  
+  # pass charts to include
+  shinyOptions(charts = charts)
```

Comment:
```suggestion
  shinyOptions(safetygraphics_charts = charts)
```

Lets make this pretty specific to our package to avoid duplicating an existing setting from elsewhere